h/claim/util.py

```diff
@@ -0,0 +1,12 @@
+def generate_claim_token(request, userid):
+    return request.registry.claim_serializer.dumps({'userid': userid})
```

Comment:
I think I'd prefer to have a function in here, `get_serializer`, that returns the serializer. It's fine that we're just stashing things on the registry object directly rather than using its API to register the serializer, but I think these details should be hidden from other code. `claim.get_serializer(request)` seems better to me.